src/content/docs/journey-migration/migrating-apm.mdx

```diff
@@ -0,0 +1,123 @@
+---
+title: "Install APM agents"
+metaDescription: "How to setup APM monitoring agents in New Relic."
+---
+
+import journeyAPMOverview from 'images/apm_screenshot-crop_overview.png'
+
+import journeyAPMPrefabAlerts from 'images/apm_screenshot-crop_precon-alerts.png'
+
+import journeyAPMDash from 'images/apm_screenshot-crop_journey-dash.png'
+
+import journeyAPMDash2 from 'images/apm_screenshot-crop_journey-dash-two.png'
+
+While setting up infrastructure monitoring provides a great foundation for understanding your systems, the best way to truly know your data comes from monitoring application performance. New Relic APM provides extensive, unified monitoring for all your applications. 
+
+That means we can better link your data through your apps and integrations, and provide more focused information for you through curated dashboards. That way, you get the level of depth and detail you need without having to hunt for what you want. And you can try out our extensive APM capabilities [absolutely free](https://newrelic.com/signup).
+
+<img
+    title="APM overview"
+    alt="An image displaying New Relic's APM monitoring page"
+    src={journeyAPMDash}
+/>
+
+## Your APM solution [#apm-solution]
+
+You may be used to `integrations` in Datadog, but in New Relic, we call these capabilities APM `agents`. We've set up our agents for both convenience and functionality. At New Relic, our agents are focused around language and linking the data they collect to other entities you've installed. 
+
+For example, an installed Java agent will collect metrics about your Java app (as well as your logs, events, and traces), and links together with an installed Ruby agent which does the same for Ruby. They link up with your infrastructure agents to provide a comprehensive, real-time view of all the data across your entire system, no matter the source.  Our agents let you:
+
+* Monitor the performance and health of your apps in one central location.
```

Comment:
Same as infra, our house style is we don't use periods after sentence-fragment-bullets. https://learn.microsoft.com/en-us/style-guide/scannable-content/lists#punctuation